### PR TITLE
Incorporate image inspect data in play kube

### PR DESCRIPTION
**Pull request by haircommander**
before, when an image signified a user, play kube ignored it. Incorporate that information.

Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

**Comment by haircommander** [comment]
fixes: #2665

File: cmd/podman/play_kube.go [hunk 1]
```diff
@@ -243,6 +244,14 @@ func kubeContainerToCreateConfig(containerYAML v1.Container, runtime *libpod.Run
 	containerConfig.Name = containerYAML.Name
 	containerConfig.Tty = containerYAML.TTY
 	containerConfig.WorkDir = containerYAML.WorkingDir
+
+	imageData, _ := newImage.Inspect(ctx)
+
+	containerConfig.User = "0"
```
Comment (TomSweeneyRedHat) [inline cmd/podman/play_kube.go]: Is this OK in a rootless scenario?  But having said that, I'm not sure what else to set it to.

**Comment by rhatdan** [comment]
LGTM

File: cmd/podman/play_kube.go (hunk 1 above)
Comment (haircommander) [reply cmd/podman/play_kube.go]: rootless is actually disabled in play kube right now. This is also the default in pkg/spec/createconfig.go 's ParseCreateConfig. So I think it's safe?

Comment (mheon) [reply cmd/podman/play_kube.go]: Yeah, rootless uses a user namespace to map UIDs, so we want to maintain the UID 0 default there too

File: cmd/podman/play_kube.go [hunk 2]
```diff
@@ -280,6 +289,7 @@ func kubeContainerToCreateConfig(containerYAML v1.Container, runtime *libpod.Run
 	for _, e := range containerYAML.Env {
 		envs[e.Name] = e.Value
 	}
+	containerConfig.Env = envs
```
Comment (mheon) [inline cmd/podman/play_kube.go]: Wait, were we just... not using the environment variables before?

Comment (haircommander) [reply cmd/podman/play_kube.go]: nope I just scooted it up because an older PR I did split the for loop for envs and setting Env

**Comment by mheon** [comment]
LGTM

**Pull request merged**